lib/include/jxl/encode.h

```diff
@@ -349,6 +394,25 @@ JxlEncoderOptionsSetEffort(JxlEncoderOptions* options, int effort);
 JXL_EXPORT JxlEncoderStatus
 JxlEncoderOptionsSetDistance(JxlEncoderOptions* options, float distance);
 
+/**
+ * Sets an option of integer type to the encoder option. The JxlEncoderOptionId
+ * argument determines which option is set.
+ *
+ * TODO(lode): also use the option enum for the container, lossless, speed,
+ * effort and distance options.
+ *
+ * @param options set of encoder options to update with the new mode.
+ * @param option ID of the option to set.
+ * @param value Integer value to set for this option.
+ * @return JXL_ENC_SUCCESS if the operation was successful, JXL_ENC_ERROR in
+ * case of an error, such as invalid or unknown option id, or invalid integer
+ * value for the given option. If an error is returned, the state of the
+ * JxlEncoderOptions object is still valid and is the same as before this
+ * function was called.
+ */
+JXL_EXPORT JxlEncoderStatus JxlEncoderOptionsSetIntegerOption(
```

Comment:
I'd suggest JxlEncoderOptionsSetInteger or JxlEncoderOptionsSetAsInteger to make it a bit less verbose; but this is fine.

Reply:
Done